gammapy/modeling/models/core.py

```diff
@@ -379,7 +379,7 @@ class DatasetModels(collections.abc.Sequence, CovarianceMixin):
 
     Parameters
     ----------
-    models : `SkyModel`, list of `SkyModel` or `Models`
+    models : `~gammapy.modelling.SkyModels`, list of `SkyModel` or `Models`
```

Comment:
```suggestion
    models : `SkyModel`, list of `SkyModel` or `Models`
```

Reply:
Why not putting the link towards the documentation? while in general all parameters are links...

Reply:
Right but here you introducing an incorrect namespace. In current doc all links are OK. See https://docs.gammapy.org/dev/api/gammapy.modeling.models.DatasetModels.html